app/src/main/java/org/schabi/newpipe/database/stream/model/StreamStateEntity.java

```diff
@@ -25,26 +23,28 @@
     // This additional field is required for the SQL query because 'stream_id' is used
     // for some other joins already
     public static final String JOIN_STREAM_ID_ALIAS = "stream_id_alias";
-    public static final String STREAM_PROGRESS_TIME = "progress_time";
+    public static final String STREAM_PROGRESS_MILLIS = "progress_time";
 
     /**
      * Playback state will not be saved, if playback time is less than this threshold.
      */
-    private static final int PLAYBACK_SAVE_THRESHOLD_START_SECONDS = 5;
+    private static final long PLAYBACK_SAVE_THRESHOLD_START_MILLISECONDS = 5000; // 5000ms = 5s
+
     /**
-     * Playback state will not be saved, if time left is less than this threshold.
+     * @see #isFinished(long)
+     * @see org.schabi.newpipe.database.feed.dao.FeedDAO#getLiveOrNotPlayedStreams()
      */
-    private static final int PLAYBACK_SAVE_THRESHOLD_END_SECONDS = 10;
+    public static final long PLAYBACK_FINISHED_END_MILLISECONDS = 60000; // 60000ms = 60s
 
     @ColumnInfo(name = JOIN_STREAM_ID)
     private long streamUid;
 
-    @ColumnInfo(name = STREAM_PROGRESS_TIME)
```

Comment:
Does this change require a Migration?

Reply:
No, I just changed the name of the variable, not its content